languages.yaml

```
@@ -1175,30 +1175,37 @@ Jsonnet:
     ansi:
       - white
       - black
-    chip: '#0064BD'
+    chip: "#0064BD"
 Jsx:
   type: programming
   ascii: |
-    {0}          .REACT.    .REACT.
-    {0}        JS     .JSXJSX.     JS
-    {0}       JS       :JSJS:       JS
-    {0}       JS     .REACTJSX'     JS
-    {0}     'JSXJSXJSXJSXJSXJSXJSXJSXJS'
-    {0}  'REACTJSREACT        REACTJSREACT'
-    {0} 'JSX'  REACT'  :JSXJ:  'REACT  'JSX'
-    {0}'JS      JSX'  :JSXJSX:  'JSX      JS'
-    {0}'JS      JSX'  :JSXJSX:  'JSX      JS'
-    {0} 'JSX'  REACT'  :JSXJ:  'REACT  'JSX'
-    {0}  'REACTJSREACT        REACTJSREACT'
-    {0}     'JSXJSXJSXJSXJSXJSXJSXJSXJS'
-    {0}       JS     .REACTJSX'     JS
-    {0}       JS       :JSJS:       JS
-    {0}        JS     .JSXJSX.     JS
-    {0}          .REACT.    .REACT.
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJS{1}XJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXSJ{1}SXJSXJS
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJS{1}XJSXJSXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSXJSXJSX
+    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSXJSXJSX
+    {0}JSXJSXJ   SX      JSXJSXJS{1}X{2}JSX{1}JSX{2}JSX{1}JSX
+    {0}JSXJSXJ   S   XJSXJSXSJSX{1}JSX{2}JSX{1}J{2}SXJ{1}SXJS
+    {0}JSXJSXJ   SXJ   SXJSXJSX{1}JSXJSX{2}JSX{1}JSXJSX
+    {0}JSXJSXJ   SXJSX   JSXJS{1}JSXJSX{2}JSXJS{1}JSXSX
+    {0}JS   XJ   S   XJ   SXJ{1}SXJSXJ{2}SXJ{1}S{2}XJS{1}XJSX
+    {0}JSX      JSX      JSX{1}JSXJSX{2}JSX{1}JSX{2}JSX{1}JSX
+    {0}JSXJSXJSXJSXJSXJSXJS{1}XJSXJSXJSXJSXJSXJSX
```

```suggestion
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJS{1}XJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXSJ{1}SXJSXJS
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJS{1}XJSXJSXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSXJ{1}SXJSXJSXJSX
    {0}JSXJSXJSXJSXJSXJSXJSXJSXJSX{1}JSXJSXJSXJSX
    {0}JSXJSXJ   SX      JSXJSXJS{1}X{2}JSX{1}JSX{2}JSX{1}JSX
    {0}JSXJSXJ   S   XJSXJSXSJSX{1}JSX{2}JSX{1}J{2}SXJ{1}SXJS
    {0}JSXJSXJ   SXJ   SXJSXJSX{1}JSXJSX{2}JSX{1}JSXJSX
    {0}JSXJSXJ   SXJSX   JSXJS{1}XJSXJS{2}XJSXJ{1}SXJSX
    {0}JS   XJ   S   XJ   SXJ{1}SXJSXJ{2}SXJ{1}S{2}XJS{1}XJSX
    {0}JSX      JSX      JSX{1}JSXJSX{2}JSX{1}JSX{2}JSX{1}JSX
    {0}JSXJSXJSXJSXJSXJSXJS{1}XJSXJSXJSXJSXJSXJSX
```
minor: fix repeating sequence

---

Fixed :+1: